docs/site/Command-line-interface.md

```diff
@@ -25,3 +25,15 @@ npm install -g @loopback/cli
 ## Generating LoopBack artifacts
 
 {% include_relative tables/lb4-artifact-commands.html %}
+
+## Naming Convention
+
+LoopBack 4 uses different convention for naming classes, variables, and files.
+
+- Class name: `PascalCase`.
+- File name: `kebabCase`.
```

Comment:
I think the example should be `kebab-case`? The current string is in camel case.

Reply:
true, I will fix it in a small PR real quick